project.clj

```diff
@@ -13,21 +13,28 @@
   ;; requires lein 2.2.0+.
   :pedantic? :abort
 
+  :exclusions [org.clojure/clojure]
```

Comment:
This is probably me not knowing what I'm doing.

This repo has a dep on Clojure 1.6, while all of our other libraries that I touched in this process depend on 1.7. Changing the version of Clojure that a library depended on seemed like a much bigger deal/possible api breaker than bumping any individual library, so instead of bumping this project to 1.7 I added this `:exclusions` like I had seen in [tk-status](https://github.com/puppetlabs/trapperkeeper-status/blob/master/project.clj#L12).

If bumping the version of Clojure is something we can do in a minor refactor then I'll do that?